manifests/init.pp

```diff
@@ -71,6 +71,24 @@
         notify  => Service[$servicename],
         content => template('dhcp/redhat/sysconfig-dhcpd'),
```

Comment:
Right, so it shouldn't be managed here - it can go into an `} else {` block below, for versions that are not EL7?